### PR TITLE
Deploy rpms SVCs on vsphere

### PR DESCRIPTION
**Pull request by hongkailiu**
Fixing https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/10294/rehearse-10294-pull-ci-openshift-installer-release-4.5-e2e-vsphere-upi/1288482492231192576

/cc @alvaroaleman @jcpowermac 

/hold

Need `secretName: mirror.openshift.com` first.

### CODE REVIEW

**Comment by alvaroaleman** [comment]
@hongkailiu could you use the chance to create a common directory, put those into that and link them into the vsphere cluster dir?

**Comment by hongkailiu** [comment]
> @hongkailiu could you use the chance to create a common directory, put those into that and link them into the vsphere cluster dir?

On it.

**Comment by alvaroaleman** [comment]
@hongkailiu have you verified that applyconfig follows symlinks?

File: Makefile [hunk 1]
```diff
@@ -240,7 +240,7 @@ build-farm-consistency:
 	@echo "diffing ns-ttl-controller assets ..."
 	diff -Naup ./core-services/ci-ns-ttl-controller/ci-ns-ttl-controller_dc.yaml ./clusters/build-clusters/01_cluster/openshift/ci-ns-ttl-controller/ci-ns-ttl-controller_dc.yaml
 	@echo "diffing rpms-ocp assets ..."
-	for file in ./core-services/release-controller/rpms-ocp-*.yaml; do diff -Naup "$${file}" "./clusters/build-clusters/01_cluster/openshift/release-controller/$${file##*/}"; done
+	for file in ./core-services/release-controller/rpms-ocp-*.yaml; do diff -Naup "$${file}" "./clusters/build-clusters/common/ocp/$${file##*/}"; done
```
Comment (alvaroaleman) [inline Makefile:243]: this is pretty useess now, we basically diff that the symlink against the symlink target?

**Comment by hongkailiu** [comment]
> @hongkailiu have you verified that applyconfig follows symlinks?
```
oc --context build01 apply -f clusters/build-clusters/common/ocp/rpms-ocp-4.1.yaml --dry-run
W0729 14:29:08.522145   34744 helpers.go:535] --dry-run is deprecated and can be replaced with --dry-run=client.
service/base-4-1 configured (dry run)
deployment.apps/base-4-1 configured (dry run)
```
At least dry-run works.

File: Makefile (hunk 1 above)
Comment (hongkailiu) [reply Makefile]: Removed.

**Comment by hongkailiu** [comment]
/override ci/prow/build-farm-consistency

The job has been removed.

**Comment by alvaroaleman** [comment]
@hongkailiu thats `oc` but we apply all of this via the `applyconfig` cli or not?

**Comment by hongkailiu** [comment]
Problem found. 

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/10545/pull-ci-openshift-release-master-vsphere-dry/1288543194274861056#1:build-log.txt%3A26

I will make changes on `applyconfig`.

**Comment by hongkailiu** [comment]
Not cool.
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/10545/pull-ci-openshift-release-master-vsphere-dry/1288841842586030080#1:build-log.txt%3A28

**Comment by hongkailiu** [comment]
/override ci/prow/build-farm-consistency

The job has been removed.

**Comment by hongkailiu** [comment]
symlink works.
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/10545/pull-ci-openshift-release-master-vsphere-dry/1288930636714741760#1:build-log.txt%3A30

@alvaroaleman 
This one is ready.

**Pull request merged**